packages/runtime-core/__tests__/apiExpose.spec.ts

```diff
@@ -216,7 +216,9 @@ describe('api: expose', () => {
     }
     const root = nodeOps.createElement('div')
     render(h(Parent), root)
+    expect('$el' in childRef.value).toBe(true)
     expect(childRef.value.$el.tag).toBe('div')
+    expect('$parent' in grandChildRef.value).toBe(true)
```

Comment:
Could you add an assertion checking for a property that is not part of publicPropertiesMap? You can do so by adding a dummy data option to the `Child` component and then asserting the content here,

Reply:
I added the assertion. Is this what you meant?

Reply:
ah, yep. perfect. I my instructions weren't precise yet that was what was needed.